guides/source/active_record_migrations.md

```diff
@@ -5,35 +5,43 @@ Active Record Migrations
 
 Migrations are a feature of Active Record that allows you to evolve your
 database schema over time. Rather than write schema modifications in pure SQL,
-migrations allow you to use a Ruby DSL to describe changes to your tables.
+migrations allow you to use a Ruby Domain Specific Language (DSL) to describe
+changes to your tables.
 
 After reading this guide, you will know:
 
-* The generators you can use to create them.
-* The methods Active Record provides to manipulate your database.
-* The rails commands that manipulate migrations and your schema.
+* Which generators you can use to create migrations.
+* Which methods Active Record provides to manipulate your database.
+* How to change existing migrations and update your schema.
 * How migrations relate to `schema.rb`.
+* How to maintain referential integrity.
 
 --------------------------------------------------------------------------------
 
 Migration Overview
 ------------------
 
-Migrations are a convenient way to
-[alter your database schema over time](https://en.wikipedia.org/wiki/Schema_migration)
-in a consistent way. They use a Ruby DSL so that you don't have to
-write SQL by hand, allowing your schema and changes to be database independent.
+Migrations are a convenient way to [evolve your database schema over
+time](https://en.wikipedia.org/wiki/Schema_migration) in a consistent way. They
```

Comment:
"Consistency" can refer to other things in databases, like data being consistent with constraint definitions.

I think the intention here is "applying the migration consistently to each Rails environment database" or something similar. Wondering if just the word alone could be improved on with more detail.

Is there a section that could be linked to that describes how schema_migrations works and how that's used to ensure versions are applied consistently?